webapp/tests/test_events.py

```diff
@@ -79,7 +80,13 @@ def test_event_tags(self):
 
         response = self.client.get(creation_url)
         self.assertEqual(response.status_code, 200)
-        self.assertRegexpMatches(response.content, re.compile('^<html>.+<title>Events</title>.+Something happened.+<td>\\[u&#39;foo&#39;, u&#39;bar&#39;\\]</td>.+</html>$', re.DOTALL))
+        if sys.version_info[0] >= 3:
+            # Python 3: 'strings'
+            expected_re = b'^<html>.+<title>Events</title>.+Something happened.+<td>\\[&#39;foo&#39;, &#39;bar&#39;\\]</td>.+</html>$'
+        else:
+            # Python 2: u'strings'
+            expected_re = b'^<html>.+<title>Events</title>.+Something happened.+<td>\\[u&#39;foo&#39;, u&#39;bar&#39;\\]</td>.+</html>$'
+        self.assertRegexpMatches(response.content, re.compile(expected_re, re.DOTALL))
```

Comment:
We should make the output consistent between the 2 versions instead, I'd be in favor of dropping the `u` prefix in py2 output